src/scripts/main.js

```diff
@@ -1,7 +1,25 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const elem = document.createElement('div');
+
+  document.body.prepend(elem);
+
+  elem.className = `notification ${type}`;
+  elem.innerHTML = `<p>${description}</p>`;
+  elem.style.top = posTop + 'px';
+  elem.style.right = posRight + 'px';
+
+  const heading = document.createElement('h2');
+
+  document.querySelector('.notification').prepend(heading);
```

Comment:
Using `document.querySelector('.notification')` will always select the first notification element in the DOM. To prepend the heading to the current notification element, use `elem.prepend(heading);` instead.